### PR TITLE
Add test to check concurrent get/create on tenant

### PR DESCRIPTION
**Pull request by xxbillouxx**
Depends-On: https://github.com/wazo-platform/xivo-dao/pull/253

### CODE REVIEW

File: integration_tests/suite/base/test_contexts.py [hunk 1]
```diff
@@ -371,3 +375,50 @@ def test_bus_events(context):
...
+
+    threads = [None] * 5
+    responses = [None] * 5
+
+    for i in range(len(threads)):
+        threads[i] = Thread(target=create_context, args=(responses, i))
+        threads[i].start()
+
+    for i in range(len(threads)):
+        threads[i].join()
```
Comment (sduthil) [inline integration_tests/suite/base/test_contexts.py:400]: How about a nicer interface like [ThreadPoolExecutor](https://www.digitalocean.com/community/tutorials/how-to-use-threadpoolexecutor-in-python-3#step-2-using-threadpoolexecutor-to-execute-a-function-in-threads)?

File: integration_tests/suite/base/test_contexts.py [hunk 2]
```diff
@@ -371,3 +375,50 @@ def test_bus_events(context):
...
+                    "response",
+                    has_property(
+                        "text",
+                        matches_regexp(
+                            r"""Unexpected error: \(psycopg2.errors.UniqueViolation\) """
+                            + r"""duplicate key value violates unique constraint .*tenant_pkey.*"""
+                        ),
+                    ),
+                )
+            )
```
Comment (sduthil) [inline integration_tests/suite/base/test_contexts.py:422]: IMHO, asserting for 500 status code is enough, we don't want *any* error to occur during this scenario.

**Review by sduthil — CHANGES_REQUESTED** [review]

**Review by sduthil — APPROVED** [review]

**Comment by sduthil** [comment]
regate

**Comment by xxbillouxx** [comment]
recheck

**Pull request merged**